src/options.cpp

```diff
@@ -456,7 +500,81 @@ bool cOpt::operator!=(const std::string sCompare) const
     return !(*this == sCompare);
 }
 
-void initOptions()
+/** Fill TILESETS mapping with values.
+ * Scans all directores in gfx directory for file named "tileset.txt".
+ * All founded values added in mapping TILESETS as name, tileset_dir.
+ * Furthermore, it builds possible values list for cOpt class.
+ * @return One string containing all found tilesets in form "tileset1,tileset2,tileset3,..."
+ */
+static std::string build_tilesets_list()
+{
+    const std::string defaultTilesets = "hoder,deon";
+
+    std::string tileset_names;
+    bool first_tileset_name = true;
+
+    TILESETS.clear();
+
+    auto const tilesets_dirs = get_directories_with(FILENAMES["tileset-conf"], FILENAMES["gfxdir"], true);
+
+    for( auto &ts_dir : tilesets_dirs ) {
+        std::ifstream fin;
+        std::string file = ts_dir + "/" + FILENAMES["tileset-conf"];
+
+        fin.open( file.c_str() );
+        if(!fin.is_open()) {
+            fin.close();
+            DebugLog( D_ERROR, DC_ALL ) << "Can't read tileset config from " << file;
+        }
+        // should only have 2 values inside it, otherwise is going to only load the last 2 values
+        std::string tileset_name;
+
+        while(!fin.eof()) {
+            std::string sOption;
+            fin >> sOption;
+
+            if(sOption == "") {
+                getline(fin, sOption);    // Empty line, chomp it
+            } else if(sOption[0] == '#') { // # indicates a comment
+                getline(fin, sOption);
+            } else {
+                if (sOption.find("NAME") != std::string::npos) {
+                    tileset_name = "";
+                    fin >> tileset_name;
+                    if(first_tileset_name) {
+                        first_tileset_name = false;
+                        tileset_names += tileset_name;
+                    } else {
+                        tileset_names += std::string(",");
+                        tileset_names += tileset_name;
+                    }
+                } else if (sOption.find("VIEW") != std::string::npos) {
+                    std::string viewName = "";
+                    fin >> viewName;
+                    optionNames[tileset_name] = viewName;
+                    break;
+                }
+            }
+        }
+        fin.close();
+        if (TILESETS.count(tileset_name) != 0) {
+            DebugLog( D_ERROR, DC_ALL ) << "Found tileset dublicate with name " << tileset_name;
+        } else {
+            TILESETS.insert(std::pair<std::string,std::string>(tileset_name, ts_dir));
+        }
+    }
+
+    if(tileset_names == "") {
+        optionNames["deon"] = _("Deon's");          // more standards
+        optionNames["hoder"] = _("Hoder's");
```

Comment:
Hmmm, those are the only translated tileset names. The other tileset go untranslated. Either they should not be translated at all (they are names anyway), or the lang/extract_json_strings.py or lang/update_pot.sh need to be changed to extract the tileset names. See also line 554, where untranslated entries are added to `optionNames`.